src/views/SpecialResolution.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <ViewWrapper>
+  <ViewWrapper v-if="loadView">
```

Comment:
I think you had a field inside of ViewWrapper that does something similar to this, but I'm afraid to change it because it will impact all filings

Reply:
Not in ViewWrapper component, but LimitedRestorationToFull.vue does something similar (but calls the variable `isDataLoaded`).

This is only needed if you can't display the page until the data is loaded. I think it's hacky so if you don't need it then please get rid of it. Otherwise, sigh, leave it.

Reply:
I need it, most of my components load in data from `created`

Reply:
ok